backend/backend/src/main/resources/example_config.yml

```diff
@@ -12,51 +26,112 @@ server:
         filterFactories:
           - type: metrics-endpoint
 
-# The JDBC (Java database connectivity) url used to connect to the database
-jdbcUrl: "jdbc:sqlite:file:data/data.db"
+##
+## Example log level configuration
+##
+## This example sets the global log level to WARN and VelCom's log level to INFO.
+## Valid log levels are: DEBUG, INFO, WARN, ERROR
+##
+#logging:
+#  level: WARN
+#  loggers:
+#    "de.aaaaaaah.velcom": INFO
 
-# The interval between listener updates. (in seconds)
-pollInterval: 120
 
-# The token for the web administrator
-webAdminToken: "12345"
+#############
+## General ##
+#############
 
-# The port that the dispatcher should be listening on
-runnerPort: 3546
+##
+## The token by which the web administrator can authenticate
+##
+webAdminToken: "12345"
 
-# The remote url where the benchmark repo remote url can be cloned from
+##
+## The URL the benchmark repo should be cloned from
+##
 benchmarkRepoRemoteUrl: "https://github.com/IPDSnelting/velcom-unibench.git"
 
-# The token runners need to provide in order to connect successfully.
-# Will be transmitted in plain text over an HTTPS connection.
-runnerToken: "Correct-Horse_Battery Staple"
+##
+## The period in which the tracked repos and bench repo is checked for updates. When new commits are
+## found, they are added to the queue.
+##
+#pollInterval: 120 # seconds
+
+
+#################
+## Directories ##
+#################
+
+##
+## VelCom organizes its files into three different directories:
+##
+## - data/  : Essential files
+## - cache/ : Non-essential files that persist between restarts
+## - tmp/   : Short-lived temporary files
+##
+## Backups only need to include the config file and the data/ dir. With those files, VelCom can
+## recreate the other two directories.
+##
+## WARNING: Files in these directories should not be changed or deleted while VelCom is running!
```

Comment:
And don't put files you hold dearly in there... Reminds me of https://xkcd.com/293/